spec/services/weworkremotely_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+
+describe WeWorkRemotely do
+  let(:service) { WeWorkRemotely }
+
+  describe '.pull_company_name' do
+    it 'pulls a company from a title' do
+      title = "Basecamp Networks: Computer Vision Engineer"
+      company_name = service.pull_company_name(title)
+
+      expect(company_name).to eq('Basecamp Networks')
+    end
+
+    it 'handles unmatching pattern' do
```

Comment:
for a test like this i sometimes like to include text describing how it handles it  (In this case i guess something like "handles unmatching pattern by leaving name blank"). Test itself is good but this helps a little bit with the "documentation" side of things